floodviz/static/js/hydrograph.js

```diff
@@ -0,0 +1,55 @@
+document.addEventListener("DOMContentLoaded", function(event) { 
+
+	"use strict";
+
+	// Collect and set hydrograph aspect ratio data
+	var hydrograph = document.getElementById('hydrograph');
+	hydrograph.style.height = FV.hydrometa['height'];
+	hydrograph.style.width = FV.hydrometa['width'];
+
+	// D3 data import
+	d3.json('../static/data/hydrograph_data.json', function(data) {
+
+		nv.addGraph( function() {
+			
+			var getX = function(d) { return d.time_mili };
+			var getY = function(d) { return d.value };
+			var getMax = function(d) { return d.max };
+
+			var min = d3.min(data, function(d) { return d3.min(d.values, getY)})
+			var max = d3.max(data, function(d) { return d3.max(d.values, getMax)})
+			
+			var chart = nv.models.cumulativeLineChart()
+					.x( getX )  // this value is stored in milliseconds since epoch (converted in data_format.py with datetime)
+					.y( getY ) 
+					.color(d3.scale.category10().range())
+					.useInteractiveGuideline(true)
+					.yDomain([min, max])
+					.margin({left: 120, top: 60})
+					.showControls(false);
+
+			chart.xAxis
+					.axisLabel(" Date (M-D-Y)")
```

Comment:
Let's follow the pattern that you established with the first chained call in line 22. So line 31 should be chart.xAxis.axLabel....